### PR TITLE
Add mobility data source

### LINKED ISSUE #69: Add mobility data source
### Description

Trello card : https://trello.com/c/N92st5EP

1.Add a data source that download the [mobility reports from Google](https://www.google.com/covid19/mobility/) and parses them.

Can use use as a starting point [this](https://gist.github.com/Amarang/3341c9a24da4556def7c3a03a12949b8#file-parse-ipynb) ( don't forget to mention on credits)

### PR DESCRIPTION
**Pull request by ManuelAlvarezC**
# Description

Add mobility data source
Fixes #69

# Checklist:

- [x] I have read the [CONTRIBUTING GUIDE](https://coronawhy.github.io/task-geo/contributing.html) and made sure this Pull Request is compliant with it.
- [x] I have read the [DATA MODEL](https://coronawhy.github.io/task-geo/data_model.html) and [DATA SOURCES](https://coronawhy.github.io/task-geo/data_sources.html) and made sure this Pull Request is compliant with it.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ManuelAlvarezC** [comment]
This finishes the work @DanielRobertNicoud started on #61. Which now will be closed.

**Comment by wwymak** [comment]
hmm, it looks like google actually now offers a csv download https://www.google.com/covid19/mobility/index.html (https://www.gstatic.com/covid19/mobility/Global_Mobility_Report.csv). Might be a bit more of a robust/accurate dataset to go with that rather than scraping pdfs? (sorry @ManuelAlvarezC I know you have spent quite a bit of time on the scrapers...)

(and you probably know the linter isn't too happy... :wink: )

File: task_geo/data_sources/mobility/audit.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+**region**
+- Description: Name of the region
+- Type: string
+
+**date**
+- Description: The date the measurements corresponds to
+- Type: date.
+
+**retail_recreation**
+- Description: Variance in the given category in relation to the baseline.
```
Comment (wwymak) [inline task_geo/data_sources/mobility/audit.md:26]:
```suggestion
- Description: Changes in movement for places classified as retail and recreation in relation to the baseline.
```
(same for the other categories-- it's a bit easier to know what's going on without going to the google mobility site)

File: task_geo/data_sources/mobility/audit.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+**transit_stations**
+- Description: Variance in the given category in relation to the baseline.
+- Type: float
+
+**workplaces**
+- Description: Variance in the given category in relation to the baseline.
+- Type: float
+
+**residential**
+- Description:
```
Comment (wwymak) [inline task_geo/data_sources/mobility/audit.md:46]: and don't forget to add a description

File: task_geo/data_sources/mobility/datapackage.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                    "description": "Name of the region",
+                    "type": "string"
+                },
+                {
+                    "name": "date",
+                    "description": "The date the measurements corresponds to",
+                    "type": "date"
+                },
+                {
+                    "name": "retai_recreation",
```
Comment (wwymak) [inline task_geo/data_sources/mobility/datapackage.json:24]:
```suggestion
                    "name": "retail_recreation",
```

File: task_geo/data_sources/mobility/datapackage.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                    "description": "Variance in the given category in relation to the baseline.",
```
Comment (wwymak) [inline task_geo/data_sources/mobility/datapackage.json:25]: same comment as above for description for the fields

**Comment by ManuelAlvarezC** [comment]
Hi @wwymak and thanks for the review!

I will start working in the changes right now!

**Pull request merged**